tests/presenters/test_show_my_plans_presenter.py

```diff
@@ -12,61 +12,6 @@
 from .url_index import HidePlanUrlIndex, PlanSummaryUrlIndexTestImpl, RenewPlanUrlIndex
 
 
-def _convert_into_plan_info(plan: Plan) -> PlanInfo:
-    return PlanInfo(
-        id=plan.id,
-        prd_name=plan.prd_name,
-        price_per_unit=Decimal("10.001"),
-        is_public_service=plan.is_public_service,
-        plan_creation_date=plan.plan_creation_date,
-        activation_date=plan.activation_date,
-        expiration_date=plan.expiration_date,
-        expiration_relative=plan.expiration_relative,
-        is_available=plan.is_available,
-        is_cooperating=bool(plan.cooperation),
-        cooperation=plan.cooperation,
-    )
-
-
-def response_with_one_plan(plan: Plan) -> ShowMyPlansResponse:
-    return ShowMyPlansResponse(
-        count_all_plans=1,
-        non_active_plans=[],
-        active_plans=[],
-        expired_plans=[],
-    )
-
-
-def response_with_one_active_plan(plan: Plan) -> ShowMyPlansResponse:
-    plan_info = _convert_into_plan_info(plan)
-    return ShowMyPlansResponse(
-        count_all_plans=1,
-        non_active_plans=[],
-        active_plans=[plan_info],
-        expired_plans=[],
-    )
-
-
-def response_with_one_expired_plan(plan: Plan) -> ShowMyPlansResponse:
-    plan_info = _convert_into_plan_info(plan)
-    return ShowMyPlansResponse(
-        count_all_plans=1,
-        non_active_plans=[],
-        active_plans=[],
-        expired_plans=[plan_info],
-    )
-
-
-def response_with_one_non_active_plan(plan: Plan) -> ShowMyPlansResponse:
-    plan_info = _convert_into_plan_info(plan)
-    return ShowMyPlansResponse(
-        count_all_plans=1,
-        non_active_plans=[plan_info],
-        active_plans=[],
-        expired_plans=[],
-    )
-
-
 class ShowMyPlansPresenterTests(TestCase):
```

Comment:
Probably we should test the corect calculation/display of the ViewModel's `expiration_relative` attribute?

Reply:
strange. I was just referring to `class ShowMyPlansPresenterTests(TestCase)`